/usr/bin/diff --ignore-matching-lines=.*{$Id:|$Header:}.* -u tcptrace.h.orig tcptrace.h
--- tcptrace.h.orig	Fri Mar  9 17:30:34 2001
+++ tcptrace.h	Fri Mar  9 17:31:08 2001
@@ -485,7 +485,7 @@
 
 /* external routine decls */
 double sqrt(double x);
-char *ether_ntoa(struct ether_addr *e);
+char *shawn_ether_ntoa(struct ether_addr *e);
 void free(void *);
 int finite(double);
 
@@ -681,7 +680,9 @@
 #define IPTOS_CE	0x01	/* Experienced Congestion */
 
 #define TH_ECN_ECHO	0x02	/* Used by receiver to echo CE bit */
+#ifndef TH_CWR
 #define TH_CWR		0x01	/* Congestion Window Reduced */
+#endif
 
 
 
